src/inference/dev_api/cpp_interfaces/interface/ie_iplugin_internal.hpp

```diff
@@ -335,6 +335,17 @@ class INFERENCE_ENGINE_API_CLASS(IInferencePlugin) : public std::enable_shared_f
     void SetExeNetworkInfo(const std::shared_ptr<IExecutableNetworkInternal>& exeNetwork,
                            const std::shared_ptr<const ov::Model>& function);
 
+    /**
+     * @brief Returns set of nodes which were removed after transformation.
+     * If originalNetwork contains node1 and transformedNetwork does not
+     * contains node1 in ops list, node1 will be returned.
+     * @param originalNetwork Original network
+     * @param transformedNetwork Transformed network
+     * @return Set of strings which contains removed node names
+     */
+    std::unordered_set<std::string> GetRemovedNodes(const CNNNetwork& originalNetwork,
```

Comment:
Shall this be used also for private plugins and other third-party plugins, like ARM?

Reply:
It can be a separate activity ONNX Runtime uses Myriad, CPU and GPU